Packs/ServiceNow/ReleaseNotes/2_7_15.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### ServiceNow Event Collector
+- Updated the Docker image to: *demisto/python3:3.12.8.3296088*.
+
+- Added support for **service-now-reset-auth** command that resets the integration context, including stored oauth refresh token. use this command when encountering access_denied or other errors during access token generation. This will let the integration generate new refresh token.
```

Comment:
Recommend:
Added support for the **service-now-reset-auth** command that resets the integration and clears the stored oauth refresh token so it can generate a new one. Use this command if you have trouble connecting due to access_denied or other errors.